From fccf8a95af1de0f435af340313a9e98c918fe787 Mon Sep 17 00:00:00 2001
From: begasus <begasus@gmail.com>
Date: Sat, 14 May 2022 17:11:12 +0200
Subject: cutter, fix check for rizin

Upstream changed the detection for rizin, for now this works

diff --git a/cmake/FindRizin.cmake b/cmake/FindRizin.cmake
index 924e112..023bff6 100644
--- a/cmake/FindRizin.cmake
+++ b/cmake/FindRizin.cmake
@@ -15,19 +15,23 @@
 #  If librz was found using find_library and not pkg-config, the following variables will also be set:
 #  Rizin_LIBRARY_<name> - Path to library rz_<name>
 
-if(WIN32)
+if(HAIKU)
 	find_path(Rizin_INCLUDE_DIRS
 			NAMES rz_core.h rz_bin.h rz_util.h
 			HINTS
 				"$ENV{HOME}/bin/prefix/rizin/include/librz"
 				/usr/local/include/libr
-				/usr/include/librz)
+				/usr/include/librz
+				/system/develop/headers/x86/librz
+				/system/develop/headers/librz)
         find_path(SDB_INCLUDE_DIR
 			NAMES sdb.h sdbht.h sdb_version.h
 			HINTS
 				"$ENV{HOME}/bin/prefix/rizin/include/librz/sdb"
 				/usr/local/include/librz/sdb
-				/usr/include/librz/sdb)
+				/usr/include/librz/sdb
+				/system/develop/headers/x86/librz/sdb
+				/system/develop/headers/librz/sdb)
 
         list(APPEND Rizin_INCLUDE_DIRS ${SDB_INCLUDE_DIR})
 
@@ -64,7 +68,9 @@ if(WIN32)
 				HINTS
 					"$ENV{HOME}/bin/prefix/rizin/lib"
 					/usr/local/lib
-					/usr/lib)
+					/usr/lib
+					/system/lib/x86
+					/system/lib)
 
 		list(APPEND Rizin_LIBRARIES ${Rizin_LIBRARY_${libname}})
 		list(APPEND Rizin_LIBRARIES_VARS "Rizin_LIBRARY_${libname}")
-- 
2.30.2

